internal/controllers/root.go

```diff
@@ -0,0 +1,30 @@
+package controllers
+
+import (
+	"database/sql"
+	"net/http"
+
+	"github.com/cyverse/QMS/internal/model"
+	"github.com/labstack/echo"
+	"gorm.io/gorm"
+)
+
+// API defines the REST API of the notifications service
```

Comment:
This comment looks like it was unmodified after being copied from the notifications service.

```suggestion
// Server defines the REST API of the QMS
```